react_frontend/src/components/Form.jsx

```diff
@@ -24,13 +27,27 @@ const Form = () => {
                 },
                 body: JSON.stringify(formData)
             });
+
             if (response.ok) {
-                console.log('Form submitted successfully');
+                let response_data = await response.json();
+                switch(response_data.message) {
+                    case 'SUCCESS':
+                        window.location.href = "/success";
```

Comment:
For window.location.href see if you can use router-react-dom module for it. Might be something like this: https://reactrouter.com/en/main/components/link

Reply:
Same for everywhere else that window.location.href is used